cellprofiler/modules/savecroppedobjects.py

```diff
@@ -87,7 +87,10 @@ def create_settings(self):
             ],
             value=O_TIFF,
             doc="""\
-**%(O_PNG)** files do not support 3D. **%(O_TIFF)** files use zlib compression level 6."""
+**{O_PNG}** files do not support 3D. **{O_TIFF}** files use zlib compression level 6.""".format(
```

Comment:
Nice catch!

Reply:
Wouldn't have caught it (and other documentation problems too- some of the added documentation was in html, not rst) if you hadn't asked me to change the formatting thing, so good on you!